src/triage/util/db.py

```diff
@@ -3,6 +3,27 @@
 import sqlalchemy
 import wrapt
 
+import json
+
+import functools
+
+def fix_data(obj):
+    """JSON serializer for objects not serializable by default json code"""
+    from psycopg2.extras import DateRange, DateTimeRange
+    from datetime import date, datetime
+
+    if isinstance(obj, (datetime, date)):
```

Comment:
`datetime` is a subclass of `date`

```suggestion
    if isinstance(obj, date):
```